cmake/test-targets.cmake

```
@@ -65,6 +65,18 @@ else ()
     message(STATUS "Set PERLCRITIC_PATH to the path of the perlcritic executable to enable Perl syntax/style checks.")
 endif ()
 
+# add test for bash script syntax
+find_program(SH_PATH shfmt)
+if (SH_PATH)
+    add_test(
+        NAME test-local-bash-syntax
+        COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tools/check-bash-scripts"
```

I guess you need to pass the path as an argument:
```suggestion
        COMMAND "${CMAKE_CURRENT_SOURCE_DIR}/tools/check-bash-scripts" "${SH_PATH}"
```

---

i just saw this but the code has already changed. I am gonna resolve it and feel free to comment in the updated commit.